src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,38 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+class FigureSupplier {
```

Comment:
The FigureSupplier class should not begin with an empty line. Please remove the empty line at the start of the class definition.